src/common/utils/position.ts

```diff
@@ -28,7 +29,7 @@ const scaleBandInvert = (scale) => {
 /**
  * Given a point position, get the closes data point in the dataset.
  */
-export const getClosestPoint = (pos: number, scale, data, attr = 'x') => {
+export const getClosestPoint = (pos: number, scale, data, attr = 'x', isPointScale=false) => {
```

Comment:
Can you add some context to what this is for ? Maybe just above the line where its used.

Reply:
added in docstring